src/main/resources/assets/minecolonies/blockstates/blockshingleslab.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
You should seriously look into forges model system. This can be made a lot smaller!.